plotly/graph_reference.py

```diff
@@ -0,0 +1,321 @@
+"""
+This module handles accessing, storing, and managing the graph reference.
+
+"""
+from __future__ import absolute_import
+
+import json
+import re
+import requests
+
+from plotly import exceptions, files, utils
+
+GRAPH_REFERENCE_PATH = '/plot-schema.json'
+META_KEYS = ['_isLinkedToArray', '_isSubplotObj', 'description', 'role']
```

Comment:
done in https://github.com/plotly/streambed/commit/81aa2710a55b50efb913efe757082d803c9aabd2